blivet/udev.py

```diff
@@ -112,23 +112,40 @@ def resolve_devspec(devspec, sysname=False):
         return ret.sys_name if sysname else device_get_name(ret)
 
 def resolve_glob(glob):
+    """ Find all devices (names) matching by path or name
+
+        :param str glob: glob to match device names or paths against
+        :returns: list of device names matching :param:`glob`
+
+    """
     import fnmatch
-    ret = []
 
-    if not glob:
-        return ret
+    paths = []
 
-    for dev in get_devices():
-        name = device_get_name(dev)
+    if glob.startswith("/dev"):
```

Comment:
Might be better to check for ``"/dev/"`` here since the code below relies on the fact that there's the ``/`` after ``/dev``